test/Microsoft.AspNetCore.WebSockets.Test/WebSocketMiddlewareTests.cs

```diff
@@ -0,0 +1,541 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace Microsoft.AspNetCore.WebSockets.Test
```

Comment:
Yeah, I can separate them. I was just copying the existing tests around and they hadn't been separated so I didn't do a lot of looking at the layout :)